frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-utils.ts

```diff
@@ -13,99 +14,169 @@ import {
   BY_EGRESS,
 } from '../../constants';
 
+/* utility function to convert number in words */
+export const numberInWords = (n: number): string => {
+  const nH = humanizeDecimalBytes(n);
```

Comment:
`humanizeNumber` is backed by `numeric` (which has only till [ '', 'k', 'm', 'b']) and at the point of writing I didn't know that it will not cross billions.

Here is why I think, we should still use `humanizeDecimalBytes` (or any other higher valued (that is; with higher unit) `humanize` function)
File 'units.js' is not something we daily change, so if we have to scale (as the requirement changes), it is still better to use a higher valued `humanize` function. The existing code will still be valid with the following change,

```typescript
const mapToWord: { [k: string]: string } = {
    B: '',
    KB: 'thousands',
    MB: 'millions',
    GB: 'billions',
}
if (mapToWord[nH.unit] == null) mapToWord[nH.unit] = `multi ${mapToWord['GB']}`;
```

Reply:
@cloudbehl , as per your suggestion, changed it to `humanizeNumber` format...